src/Backend/Modules/Analytics/Installer/Uninstaller.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Backend\Modules\Analytics\Installer;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+
+use Backend\Core\Installer\ModuleUninstaller;
+use Backend\Core\Installer\UninstallerInterface;
+
+/**
+ * Uninstaller for the analytics module
+ */
+class Uninstaller extends ModuleUninstaller implements UninstallerInterface
+{
+    public function uninstall(): void
+    {
+        $this->setModule('Analytics');
+
+        $this->deleteBackendNavigation();
+        $this->deleteBackendWidgets();
+
+        $this->dropModule();
+    }
+
+    private function deleteBackendNavigation(): void
+    {
+        $this->deleteNavigation('Settings.Modules.' . $this->getModule());
```

Comment:
why do you use a . notation? and not just /settings/modules/analytics for instance?

Reply:
It does not matter, through the slashes to implement. To make the slashes?

Reply:
I think since we don't have routes with . names slashes that actually match the url in the backend make more sense